features/step_definitions/cms/section_steps.rb

```diff
@@ -21,11 +21,11 @@
 end
 
 def add_section_to_provider(provider, protection, name, path = nil)
-  root = provider.sections.root || FactoryBot.create(:root_cms_section, :account => provider)
+  root = provider.sections.root || FactoryBot.create(:root_cms_section, provider: provider)
 
   options = {:title => name,
     :system_name => name,
-    :account => provider,
+    :provider => provider,
```

Comment:
Maybe while on it, change the syntax of the whole hash?

Reply:
I thought it was not directly related and could make the review harder